migrations/versions/888b56ed5dcb_.py

```diff
@@ -10,12 +10,28 @@
 revision = '888b56ed5dcb'
 down_revision = 'd415d490eb05'
 
-from alembic import op
+from alembic import op, context
 import sqlalchemy as sa
+from sqlalchemy.schema import Sequence, CreateSequence
+
+
+def dialect_supports_sequences():
+    migration_context = context.get_context()
+    return migration_context.dialect.supports_sequences
+
+
+def create_seq(seq):
+    if dialect_supports_sequences():
+        op.execute(CreateSequence(seq))
 
 
 def upgrade():
     try:
+        seq = Sequence('customuserattribute')
```

Comment:
The sequence should always be named "customeruserattriute_seq", cincludeing _seq.